Created plate protection layer for tents

MaxDac opened the pull request:
Created a plate protection layer to guarantee more exposed bottom surface in tent design. The protection layer can be printed using transparent plastic, resin, or other material to let the LED light pass without interference.

In the following an example image:

![plate-protection-layer](https://user-images.githubusercontent.com/25101798/219360071-d1c29ae9-a61b-427c-83df-4f804a58ffa9.jpg)

bstiq commented:
Hey, 

I'm not sure what's the point of this.
To diffuse the light ? There is no underglow RGB on the TBK Mini.

MaxDac commented:
> Hey,
> 
> I'm not sure what's the point of this. To diffuse the light ? There is no underglow RGB on the TBK Mini.

Not exactly, I build this just to create a layer below the keyboard for the new 30-degree tent. It's meant as a protection layer for the bottom part of the keyboard internal when using tents that leave a lot of open space at the bottom. I printed it transparent just because I thought there was an underglow to the bottom.

I create the PR just to check whether it's worth adding it to the files, but feel free to reject it if you deem this model unnecessary.

bstiq commented:
Ah, fair. 
It's nice to have I guess, could you maybe move it to the `tents` folder and add some info in the readme ?
See my previous commit for reference, I got it wrong but you can get an idea of where to place it from there.

MaxDac commented:
> Ah, fair. It's nice to have I guess, could you maybe move it to the `tents` folder and add some info in the readme ? See my previous commit for reference, I got it wrong but you can get an idea of where to place it from there.

I moved the file (it actually made sense to move it there, having that its use case would be an addition to those) and added the line to the readme, I hope it's understandable enough, otherwise feel free to change the description of the file. I'm currently printing them again in orange, I can send more descriptive photos if you want.
